### ISSUE TITLE
Don't override global.Promise

### ISSUE THREAD

**Issue by maximilianschmitt** [marcbachmann/node-html-pdf#188]
```javascript
console.log(Promise)
console.log('requiring html-pdf')
const pdf = require('html-pdf')
console.log(Promise)
```

Outputs:

```
[Function: Promise]
requiring html-pdf
{ [Function: lib$es6$promise$promise$$Promise]
  all: [Function: lib$es6$promise$promise$all$$all],
  race: [Function: lib$es6$promise$promise$race$$race],
  resolve: [Function: lib$es6$promise$promise$resolve$$resolve],
  reject: [Function: lib$es6$promise$promise$reject$$reject],
  _setScheduler: [Function: lib$es6$promise$asap$$setScheduler],
  _setAsap: [Function: lib$es6$promise$asap$$setAsap],
  _asap: [Function: asap] }
```

**Comment by marcbachmann** [comment]
This library doesn't even use Promises 😢 
It's depending on `phantomjs-prebuilt`, which uses `es6-promise`, which replaces global.Promise on node 4. This doesn't happen in node 6.

I've created a pr here: https://github.com/Medium/phantomjs/pull/625
I guess the update will land in here within the next few days.

**Comment by maximilianschmitt** [comment]
Hey @marcbachmann! Awesome, thanks for the fast response!

**Comment by tzi** [comment]
Since the fix is published in phantomjs-prebuilt v2.1.13, we can certainly close this issue :dancer:

**Comment by maximilianschmitt** [comment]
Thanks for fixing this so quickly. 👍  :)
